rbi/annotations/stripe.rbi

```diff
@@ -18,33 +21,40 @@ class Stripe::APIResource < Stripe::StripeObject
   # @method_missing: from StripeObject
   sig { params(val: T::Hash[T.any(String, Symbol), T.untyped]).void }
   def metadata=(val); end
-
-  sig { params(id: T.any(String, T::Hash[Symbol, T.untyped]), opts: T.nilable(T::Hash[Symbol, T.untyped])).returns(Stripe::APIResource) }
-  def self.retrieve(id, opts = nil); end
 end
 
 class Stripe::ApplicationFee < Stripe::APIResource
-  Elem = type_member {{fixed: T.untyped}}
+  Elem = type_member { { fixed: T.untyped } }
+
+  sig { params(id: T.any(String, T::Hash[Symbol, T.untyped]), opts: T.nilable(T::Hash[Symbol, T.untyped])).returns(Stripe::ApplicationFee) }
+  def self.retrieve(id, opts = nil); end
 
   # @method_missing: from StripeObject
   sig { returns(String) }
   def id; end
+
+  # @method_missing: from StripeObject
+  sig { returns(Stripe::ListObject) }
+  def refunds; end
```

Comment:
I couldn't tell where `refunds` and `reversals` are coming from. Loading [triggers](https://github.com/stripe/stripe-ruby/blob/e4e6bb54fdb53aef1edf8e92a14508abc61a3fa1/lib/stripe/resources/application_fee.rb#L11) `nested_resource_class_methods`. Which seems to [define](https://github.com/stripe/stripe-ruby/blob/e4e6bb54fdb53aef1edf8e92a14508abc61a3fa1/lib/stripe/api_operations/nested_resource.rb#L38) the methods prefixed with operations and a [method suffixed](https://github.com/stripe/stripe-ruby/blob/e4e6bb54fdb53aef1edf8e92a14508abc61a3fa1/lib/stripe/api_operations/nested_resource.rb#L22) with `_url`.